src/main/webapp/app/exercises/text/assess/text-submission-assessment.component.html

```diff
@@ -60,7 +60,7 @@
 </ng-template>
 
 <ng-template #noSubmission>
-    <div *ngIf="">
+    <div *ngIf="!loadingInitialSubmission">
         <div *ngIf="noNewSubmissions" class="alert alert-warning text-center mt-4" role="alert">
```

Comment:
the if can be moved updwards to remove the other one

```suggestion
    <div *ngIf="!loadingInitialSubmission && noNewSubmissions">
        <div class="alert alert-warning text-center mt-4" role="alert">
```

Reply:
That doesn't work because there is a paragraph with `*ngIf="!noNewSubmissions"` in the same div!